internal/testing/ci/smoke/smoke_test.go

```diff
@@ -0,0 +1,96 @@
+//go:build smoke
+
+package smoke
+
+import (
+	"os"
+	"testing"
+
+	"github.com/alecthomas/kong"
+	"github.com/circleci/ex/config/secret"
+)
+
+type CLI struct {
+	CircleToken   secret.String `name:"circle-api-token" env:"CIRCLE_API_TOKEN" required:"true" help:"An API token to authenticate with the CircleCI API."`
+	TriggerSource string        `name:"trigger-source" env:"CIRCLE_BUILD_URL" default:"dev" help:"Where this pipeline was triggered from."`
+	T             string        `name:"smoke.test" short:"t"`
+
+	Tests struct {
+		Branch   string `name:"branch" env:"BRANCH" default:"main" help:"Which branch to run the tests on."`
+		Version  string `name:"version" env:"VERSION" default:"edge" help:"The runner agent version to use in the tests."`
+		IsCanary bool   `name:"is-canary" env:"IS_CANARY" default:"false" help:"Whether this is a canary or not. Some things like the Docker image repositories may differ for canaries."`
+
+		// Driver-specific parameters
+		Kubernetes `envprefix:"KUBERNETES_"`
+	} `envprefix:"SMOKE_TESTS_" embed:""`
+}
+
+type Kubernetes struct {
+	HelmChartBranch string `env:"HELM_CHART_BRANCH" default:"" help:"An optional branch name on the CircleCI-Public/container-runner-helm-chart repository. This can be used for testing a pre-release Helm chart version."`
+}
```

Comment:
Added this extra option to match the available pipeline parameter for this: https://github.com/circleci/runner-smoke-tests/blob/65e76930ddea3fd7ba5e901aee00de8fb8e0fbf1/.circleci/config.yml#L32